resources/documentation/multiple_icebergs.rst

```diff
@@ -227,45 +291,226 @@ And add management2 to iceberg2, by editing /etc/bluebanquise/inventory/cluster/
   iceberg_level = 2
 
   [iceberg2]
-  management2
+  mngt2
+
+Play again playbooks on mngt1, so mngt2 is added into dhcp, pxe, dns, hosts,
+etc. configuration files.
+
+.. note::
+  Even if mngt2 is not part of iceberg1, it has been added to configuration
+  files on mngt1, like any other nodes of iceberg1. All nodes part of a sub
+  iceberg and also part of *mg_managements* group are automatically added, as
+  they also need to be deployed from this iceberg, like any other nodes.
+
+Once done, use standard procedure to deploy OS on mngt2 from mngt1 (*bootset*,
+etc).
+
+Now, few steps has to be followed in a specific order in order to deploy
+configuration on mngt2.
+
+Deploy sub management configuration
+-----------------------------------
+
+First, using default strategy (you can use another one), it is needed that mngt2
+mount over nfs the repositories and the BlueBanquise configuration from mngt1.
+This to be able to install packages, but also act as a repository server for
+its iceberg, and be able to deploy the configuration on its iceberg nodes.
+
+We need to ensure mngt2 is part of a group that will mount the repositories and
+bluebanquise, in nfs.yml. By default, this group is called
+*secondary_managements*.
+
+Create file *inventory/cluster/groups/secondary_managements* with the following
+content:
+
+.. code-block:: text
+
+  [secondary_managements]
+  mngt2
+
+Then ensure in file *inventory/group_vars/all/general_settings/nfs.yml* you have
+exactly at least these two exports:
```

Comment:
*exactly at least* is wrong. ...you have at least these... ?

Reply:
done